### PR TITLE
feat : Signup Page Added

### PR DESCRIPTION
**Pull request by vikrantkode**
# Open For Reviews

### CODE REVIEW

**Review by Koushith — APPROVED** [review]
can make use of semantics- section, nav, etc. rest of the things are good to go.

File: Pages/Signup/signup.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                        <h3>Signup</h3>
+                    </div>
+                    <div class="input_textbox ">
+                        <label class="form_label">Email Address:</label>
+                        <input class="input_box login_textbox" type="text" required
+                            placeholder="vikrant2812@gmail.com" />
+                    </div>
+                    <div class="input_textbox ">
+                        <label class="form_label">password:</label>
+                        <input class="input_box login_textbox" type="email" required placeholder="**********">
```
Comment (anurag-kkumarr) [inline Pages/Signup/signup.html]: check type attribute in both email and password.

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]

**Review by anurag-kkumarr — CHANGES_REQUESTED** [review]
Else everything looks good

File: Pages/Signup/signup.html (hunk 1 above)
Comment (vikrantkode) [reply Pages/Signup/signup.html]: changed type="password"

**Comment by vikrantkode** [comment]
> can make use of semantics- section, nav, etc. rest of the things are good to go.

Will make use of that during refactoring.

**Pull request merged**